src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -124,7 +124,7 @@ internal static object GetPrimitiveDefaultValue(Type type)
             switch (type.GetTypeCode())
             {
                 case TypeCode.Boolean:
-                    result = ScriptingRuntimeHelpers.False;
+                    result = False;
```

Comment:
To make it consistent with the `Int32_` prefixed boxed values, adding a `Boolean_` prefix to these for clarity.